src/components/Carousel/Carousel.js

```diff
@@ -30,19 +30,20 @@ const MyCarousel = () => {
   const [data, setData] = React.useState(null);
   React.useEffect(()=>{
     getData().then((data) => setData(data));
-  });
+  }, []);
```

Comment:
don't forget to do this if you only want the effect to run on the initial render